src/components/Icon/Icon.jsx

```diff
@@ -118,6 +135,7 @@ const Icon = createClass({
 					},
 					className
 				)}
+				onClick={this.handleSelect}
```

Comment:
this.handleSelect isn't defined, so this would cause an error. it looks like `this.onSelect` is defined above but also consider renaming that method to `this.handleClick` for consistency